src/ui/components/pages/contacts/searchResults/index.js

```diff
@@ -65,27 +65,50 @@ export const ContactsSearchResults = ({ formValues }) => {
 
   return (
     <>
-      <h2>Résultats</h2>
-      <p>{`Cliquez sur l'identifiant du contact pour consulter ou modifier ses coordonnées.`}</p>
-      <div style={{ height: 400, width: "90%" }}>
-        <DataGrid
-          localeText={frFR.components.MuiDataGrid.defaultProps.localeText}
-          columns={columns}
-          getRowId={row => pageState.data.indexOf(row)}
-          rows={pageState.data}
-          rowCount={pageState.total}
-          loading={pageState.isLoading}
-          rowsPerPageOptions={[5, 10, 50]}
-          pagination
-          page={pageState.page - 1}
-          pageSize={pageState.pageSize}
-          paginationMode="server"
-          onPageChange={newPage => {
-            setPageState(old => ({ ...old, page: newPage + 1 }));
-          }}
-          onPageSizeChange={newPageSize => setPageState(old => ({ ...old, pageSize: newPageSize }))}
-        />
-      </div>
+      (<Typography variant="h6">Nombre de résultats: {pageState.total}</Typography>
+      {pageState.data.map(c => (
+        <>
+          <Accordion
```

Comment:
When you generate a list of item, React want to you to add `key` props to elements.
For exemple, you can do :
```js
{pageState.data.map(c => (
        <Accordion
          key={`${c?.identifier}-accordion`}
          ...>
          {...}
         </Accordion>
))}
```